_data/apis.json

```diff
@@ -14866,5 +14866,525 @@
+            "fi",
+            "fr",
+            "he",
+            "hi",
+            "hr",
+            "hu",
+            "id",
+            "it",
+            "ja",
+            "ko",
+            "lt",
+            "lv",
+            "ms",
+            "nl",
+            "no",
+            "pl",
+            "pt",
+            "ro",
+            "ru",
+            "sk",
+            "sl",
+            "sv",
+            "th",
+            "tr",
+            "uk",
+            "vi",
+            "zh"
+        ],
+        "inputs": [
+            "text"
+        ],
+        "self-serve": true,
+        "models": [
+            "TowerLLM"
+        ]
+    },
+    {
+        "name": "Lara",
+        "id": "lara",
+        "company": "translated",
+        "urls": [
+            "https://translated.com/lara",
+            "https://developers.laratranslate.com/docs/introduction",
+            "http://support.laratranslate.com"
+        ],
+        "languages": [
+            "ar",
+            "bg",
+            "ca",
+            "cs",
+            "da",
+            "de",
+            "el",
+            "en-au",
+            "en-ca",
+            "en-gb",
+            "en-ie",
+            "en-us",
+            "es-419",
+            "es-ar",
+            "es-es",
+            "es-mx",
+            "fi",
+            "fr-ca",
+            "fr-fr",
+            "he",
+            "hr",
+            "hu",
+            "id",
+            "it",
+            "ja",
+            "ko",
+            "ms",
+            "nb",
+            "nl-be",
+            "nl-nl",
+            "pl",
+            "pt-br",
+            "pt-pt",
+            "ru",
+            "sk",
+            "sv",
+            "th",
+            "tr",
+            "uk",
+            "zh-cn",
+            "zh-hk",
+            "zh-tw"
+        ],
+        "inputs": [
+            "text"
+        ],
+        "formality": true,
+        "adaptive": true,
+        "self-serve": true
+    },
+    {
+        "name": "Sarvam AI",
+        "id": "sarvam-ai",
+        "company": "sarvam-ai",
+        "urls": [
+            "https://docs.sarvam.ai",
+            "https://sarvam.ai"
+        ],
+        "languages": [
+            "as-in",
+            "bn-in",
+            "brx-in",
+            "doi-in",
+            "en-in",
+            "gu-in",
+            "hi-in",
+            "kn-in",
+            "kok-in",
+            "ks-in",
+            "mai-in",
+            "ml-in",
+            "mni-in",
+            "mr-in",
+            "ne-in",
+            "or-in",
+            "pa-in",
+            "sa-in",
+            "sat-in",
+            "sd-in",
+            "ta-in",
+            "te-in",
+            "ur-in"
+        ],
+        "inputs": [
+            "text",
+            "speech",
+            "file"
+        ],
+        "formality": true,
+        "self-serve": true,
+        "models": [
+            "Sarvam-Translate"
+        ]
+    },
+    {
+        "name": "Qwen3‑MT",
```

Comment:
I think we should just call it Qwen-MT / `qwen-mt`.

That's what they call it.

The model version will change too often.